testing/integration/k8s/k8s.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package k8s
+
+import "path/filepath"
+
+const (
+	AgentKustomizePath = "./testdata/elastic-agent-kustomize.yaml"
+	AgentHelmChartPath = "../../../deploy/helm/elastic-agent"
```

Comment:
maybe we can try to be consistent in using `filepath` package with relative paths like it's done for `KubeStackChartPath` ?